main.go

```diff
@@ -144,12 +145,6 @@ func wrappedMain() int {
 
 	inPlugin := os.Getenv(plugin.MagicCookieKey) == plugin.MagicCookieValue
 
-	// Prepare stdin for plugin usage by switching it to a pipe
-	// But do not switch to pipe in plugin
-	if !inPlugin {
-		setupStdin()
```

Comment:
I'm now wondering if removing that call could also allow to remove the `github.com/mattn/go-tty` import/usage 🤔

Reply:
FYI: nope !